app/src/main/java/org/covidwatch/android/domain/EnableContactEventLoggingUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package org.covidwatch.android.domain
+
+import android.content.SharedPreferences
+
+private const val IS_CONTACT_EVENT_LOGGING_ENABLED_KEY = "preference_is_contact_event_logging_enabled"
+
+class EnableContactEventLoggingUseCase(
+    private val preferences: SharedPreferences
+) {
+
+    fun execute() {
+        val isContactLoggingEnabled: Boolean = preferences.getBoolean(IS_CONTACT_EVENT_LOGGING_ENABLED_KEY, false)
```

Comment:
We do not set `preference_is_contact_event_logging_enabled` to be `true` anywhere in the code we won't start the service. 

In general I don't like this logic of setting a key in SP and doing something after changing it. We should think of better ways. But this will do for now.

Reply:
Fixed